config/jest/setupTests.tsx

```diff
@@ -35,13 +35,12 @@ jest.mock("react-i18next", () => ({
       changeLanguage: () => new Promise(() => {}),
     },
   }),
-  Trans: ({ children }: { children: React.ReactElement[] }) =>
-    children.map((child, i) => {
-      if (typeof child === "string") {
-        return <span key={i}>{child}</span>;
-      }
-      return "";
-    }),
```

Comment:
just curious - why was this necessary?

Reply:
So it looks to me like this was incorrect but no test had hit this path to exercise it yet. In every case that we use Trans, we use a string as its child so expecting an array broke the test I was writing as it tried to render that warning.